command/registry.go

```diff
@@ -240,6 +242,8 @@ func RegisteredCommands(ui cli.Ui) map[string]mcli.CommandFactory {
 		entry{"tls ca create", func(ui cli.Ui) (cli.Command, error) { return tlscacreate.New(ui), nil }},
 		entry{"tls cert", func(ui cli.Ui) (cli.Command, error) { return tlscert.New(), nil }},
 		entry{"tls cert create", func(ui cli.Ui) (cli.Command, error) { return tlscertcreate.New(ui), nil }},
+		entry{"troubleshoot connect", func(ui cli.Ui) (cli.Command, error) { return troubleshoot.New(ui), nil }},
```

Comment:
If you follow the pattern of the other commands, there is usually an entry at the subcommand level to provide intermediate help and usage guideline, e.g. [tls.go](https://github.com/hashicorp/consul-enterprise/blob/eaca8f33f71da474d45dccddcbf1ce3f027b1ee4/command/tls/tls.go#L10-L10)